composer.json

```diff
@@ -17,20 +17,27 @@
         }
     ],
     "require": {
-        "php": ">=5.3.6",
+        "php": "^7.1",
         "behat/behat": "^3.0",
         "behat/symfony2-extension": "^2.0",
-        "doctrine/data-fixtures": "^1.0"
+        "doctrine/data-fixtures": "^1.0",
+        "doctrine/doctrine-bundle": "^1.8",
```

Comment:
is it good idea to put bundles as mandatory options? isn't behat used outside symfony infrastructure?

Reply:
Makes sense. Done.